spec/views/users_index_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.feature 'User index page', type: :feature do
+  before do
+    visit users_path
+  end
+
+  scenario 'Displays name, profile picture, and post count for all users' do
+    sleep(1)
+    expect(page).to have_content('Clement')
+    expect(page).to have_css("img[src='https://avatars.githubusercontent.com/u/53082877?s=96&v=4']")
+    expect(page).to have_css('li.user div.post-counter-details p', text: 'Number of posts: 0')
+    expect(page).to have_css('li.user', count: 3)
+  end
+
+  scenario 'Clicking on a user redirects to their show page' do
+    user = User.find_by(name: 'Clement')
+    click_on 'Clement'
+    sleep(1)
+    expect(current_path).to eq(user_path(user.id))
+  end
+end
```

Comment:
As shown in the image below, some of the tests are failing, kindly check and ensure that all the tests passed in order to produce a good and up to standard app. 
NB: this applies to all the tests in the app.


![Screenshot from 2023-03-08 12-00-43](https://user-images.githubusercontent.com/65233860/223709335-cfe82df1-57ef-44ec-8055-ab77ab47a701.png)


![Screenshot from 2023-03-08 12-01-01](https://user-images.githubusercontent.com/65233860/223709352-4cd2e352-cc92-44c1-980b-4abe39302535.png)

Reply:
hi @jerryowusu, I think the requirement of this project is to write integration tests which I did in the spec/views folder

![image](https://user-images.githubusercontent.com/53082877/223723235-090b5f10-0e67-46a0-9ee1-9f4dcf346804.png)